Tests/MongoSwiftSyncTests/SyncTestUtils.swift

```diff
@@ -63,6 +63,16 @@ extension MongoClient {
         return try ServerVersion(versionString)
     }
 
+    /// Determine whether server version and topology requirements for a certain test are met
+    internal func meetsRequirements(_ testRequirement: TestRequirement) throws -> UnmetRequirements? {
+        let reply = try self.db("admin").runCommand(["isMaster": 1])
+        return try commonMeetsRequirements(
+            testRequirement: testRequirement,
+            serverVersion: serverVersion(),
```

Comment:
As part of the driver's style, we require explicit `self` to signal where method calls are going to. For here that means doing `self.serverVersion()`.

Reply:
done